### DISCUSSION TITLE
Fiber count matrix divided by the number of voxel-voxel pairs

### DISCUSSION THREAD

**Discussion by leotozzi88** [neurolib-dev/neurolib#199] [Q&A] [answered]
Dear Experts, 

Sorry for bugging you again! 

I have received access to some diffusion matrices from the Human Connectome Project for use in my analyses. Unfortunately the fiber count matrix is not in any of the three formats you recommend in your paper. What they did is similar to the "nvoxel" normalization method implemented in your library, but a bit different. Instead of dividing the total number of fibers in each column of the count matrix by the size of each region, they divided each element of the matrix by the sum of the number of voxels of each region pair. In this way, the matrix obtained is symmetric. A similar thing was done for the distance.

I would like to use these matrices for my analyses, but I wanted to check with you if they are compatible with your algorithms, since they are not normalized in a way implemented in your library. I would welcome any advice on how to proceed!

Thank you very much, 

Leonardo Tozzi

**Comment by caglorithm** [comment, marked as answer]
Hi Leonardo,

I see no problem in using differently normalized connectivity matrices with neurolib, as long as you are aware what the specific effect of a normalization method is. You can simply pass your custom matrices at model creation:

```python
model = ALNModel(Cmat = custom_Cmat, Dmat = custom_Dmat)
```

You can check the matrices by calling `model.params['Cmat']`. 

You might have to adjust the global coupling strength `model.params['Ke_gl']` to a suitable value, depending on how large the entries of your connectivity matrices are, and, correspondingly, adjust `model.params['signalV']` to values suitable for your delay matrix.

Cheers!

**Reply by leotozzi88** [reply]
Thank you very much!
